src/Microsoft.Security.Utilities.Core/DataClassification/DAT101_000_FormalGuid.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class FormalGuid : RegexPattern
```

Comment:
>FormalGuid [](http://example.com/codeflow?start=17&length=10)

A formal GUID on research turns out to be a `UUID`

so let's rename this `Uuid` and we can create another rule for that apparent `Guid` spillover.